src/vcl/tracer.js

```diff
@@ -43,7 +54,7 @@ function logvars(start) {
     return data;
   }, {});
 
-  return usedvars;
+  return {} || usedvars;
```

Comment:
isn't this the wrong way around?

also see https://lgtm.com/projects/g/adobe/helix-publish/rev/pr-f2960fda16d80fa30e66624dd7181d25f30b1617